activejob/lib/active_job/test_helper.rb

```diff
@@ -77,14 +77,23 @@ def queue_adapter_for_test
     #       HelloJob.perform_later('jeremy')
     #     end
     #   end
-    def assert_enqueued_jobs(number, only: nil)
+    #
+    # If a queue name is passed, the number of times jobs enqueued to that specific queue can be asserted.
```

Comment:
I think this sentence is confusing and can be worded better. How about something like "Asserts jobs were enqueued the specified number of times. If a queue name is passed, asserts jobs were enqueued for that specific queue."